range-slider/src/app/app.component.ts

```diff
@@ -11,8 +12,20 @@ export class AppComponent {
   public someMin: number = -10;
   public someMax: number = 10;
 
+  @ViewChild('slider') slider: NouisliderComponent;
+
   onChange(value: any) {
     console.log('Value changed to', value);
+    this.someValue = [value[0], this.someValue[1]];
+  }
```

Comment:
Does it really make sense to treat `value` as an array? I guess you could input a number through the HTML input elements and assign it to the dedicated entry in `someValue`.